mailpoet/assets/css/src/components-automation-editor/_panel.scss

```diff
@@ -20,3 +20,35 @@
     padding: 4px;
   }
 }
+
+.mailpoet-step-name-dropdown {
+  display: block;
+
+  h2 {
+    margin: 0;
+  }
+
+  .mailpoet-automation-panel-plain-body-title-text {
+    padding-left: 0;
+    padding-top: 0;
+  }
+
+  .mailpoet-automation-panel-plain-body-title-action {
+    margin-top: -13px;
+    padding-right: 0;
+    padding-top: 0;
+  }
```

Comment:
The figma shows a blue outline around this element in the active state, which I'm not seeing in my local.
## Figma
<img width="93" alt="image" src="https://user-images.githubusercontent.com/8656640/184236423-1117d903-448a-4e87-b8de-1138cbd51e15.png">

## Local
<img width="45" alt="image" src="https://user-images.githubusercontent.com/8656640/184236465-af721b1d-7a01-41c1-b160-7dec66560107.png">

Reply:
It is in the `focus` state. I checked now with the Gutenberg editor. There we have the same behavior. Blue on focus but not on active state. I think, we should follow the precedence. 
![image](https://user-images.githubusercontent.com/6458412/184294162-2d7b146e-dce7-4fbf-84d9-1091f3f40d1b.png)

Reply:
That makes total sense! I imagine that's what was intended, but even if not consistency seems like what we should be shooting for.